ocs_ci/ocs/ui/views.py

```
@@ -2165,6 +2165,11 @@
         'span[data-test="{}-replicas"]',
         By.CSS_SELECTOR,
     ),
+    "block_pool_replica": (
+        "//dt[normalize-space()='Data protection policy']"
+        "/following-sibling::dd[@data-test='detail-item-value']",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)